src/lazy_bdecode.cpp

```diff
@@ -436,7 +436,7 @@ namespace {
 		else if (int(m_size) == this->capacity())
 		{
 			int const capacity = this->capacity() * lazy_entry_grow_factor / 100;
```

Comment:
Perhaps capacity should be size_t here too. There is a miltiplication, so widening might make sense

Reply:
OK